src/Form/FilePersisterServiceSettingsForm.php

```diff
@@ -47,7 +47,12 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#required' => TRUE
 
     ];
-
+    $form['file_path'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Relative Path for Persisting Files'),
+      '#description' => $this->t('Path relative to the root of the storage scheme selected above where the hashed directories used to store persisted managed files will be created. Do not include beginning or ending slashes. Default is "".'),
+      '#default_value' => !empty($config_storage->get('file_path')) ? $config_storage->get('file_path'): "",
```

Comment:
Wonder if we should better go for NULL instead of empties? And then change the logic of the setting? Not sure really

Reply:
My thinking was that the user input for this field is trimmed, exploded on slashes, etc - all operations that expect a string.

Reply:
Ok, let's roll with an empty string